src/main.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+import './style/main.scss';
+
+const apiUrl = 'https://www.reddit.com/r';
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      thingSearchedFor: '',
+      searchLimit: '',
+    };
+
+    // this.handlePokemonNameChange = this.handlePokemonNameChange.bind(this);
+    this.searchFormBoard = this.searchFormBoard.bind(this);
+    // this.searchFormLimit = this.searchFormLimit.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  searchFormBoard(event) {
+    this.setState({ thingSearchedFor: event.target.value });
+  }
+
```

Comment:
need another event handler to change `this.state.limit` based on the input value